backend/src/main/java/com/huellapositiva/domain/service/ProposalService.java

```diff
@@ -325,4 +328,38 @@ public void changeStatusToPublishedFromEnrollmentClosed(String proposalId) {
                 .name("PUBLISHED").build();
         jpaProposalRepository.updateStatusById(proposalId, jpaProposalStatus);
     }
+
+    @Scheduled(cron = "0 ${random.int[0,59]} 0 * * ?")
+    public void changeExpiredProposalStatusToFinished() throws ParseException {
```

Comment:
This test does not throw `ParseException` now, it is being marked as a code smell in the Sonar report. Have a look at the report to fix 2 out of the 3 code smells marked. You can ignore the naming convention one.

Reply:
Crap, I forgot to check Sonar after last refactor. Sorry!